emails/templates/ng_alert_notification.txt

```diff
@@ -1,38 +1,60 @@
-[[HiddenSubject .Subject "[[.Title]]"]]
```

Comment:
I _think_ removing the `HiddenSubject` is sensible since there is no `<title>` tag when using `text/plain`

Reply:
I tested it with and without and as far I I can tell it is a no-op.